packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -61,19 +70,53 @@ export class ConnectionStateHandler extends EventEmitterWithErrorHandling<IConne
         super();
     }
 
-    public receivedAddMemberEvent(clientId: string, quorum: Quorum) {
+    // This is true when this client submitted any ops.
+    public setDirtyState() {
+        this.isDirty = true;
```

Comment:
I do not see any place where it becomes false while connection is active

Reply:
This is just to get informed that this client has sent any ops. So I have set it to false when we get Connected as the connection is new from there. Maybe I should change the name of function.

Reply:
Changed the function name.